@kiva/kv-components/src/vue/KvLendCta.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<div class="tw-whitespace-nowrap">
 		<kv-cart-pill
```

Comment:
We actually want to remove `KvCartPill` entirely, since we no longer use this component.